editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+using TOpeningDays = set<osmoh::Weekday>;
```

Comment:
It's not a constant, it's a typedef. And you cannot guarantree that someone will not create a typedef with the same name in the same namespace somewhere.